boms/cloud-lts-bom/pom.xml

```diff
@@ -57,7 +57,7 @@
     <google.cloud.spanner.version>6.0.0</google.cloud.spanner.version>
     <google.cloud.bigtable.version>1.21.2</google.cloud.bigtable.version>
     <google.cloud.bigtable.client.version>1.19.1</google.cloud.bigtable.client.version>
-    <google.autovalue.version>1.8.0</google.autovalue.version>
```

Comment:
I noticed that correct value for 1.8.0 release was "1.8". This reminded me that we don't have any validation to this file yet.

<img width="702" alt="Screen Shot 2021-04-22 at 16 15 24" src="https://user-images.githubusercontent.com/28604/115779485-0cc34100-a386-11eb-80d8-4a4c13ad9e19.png">